web/src/bootstrap_typeahead.ts

```diff
@@ -173,15 +173,15 @@ import * as tippy from "tippy.js";
 import * as scroll_util from "./scroll_util";
 import {get_string_diff, the} from "./util";
 
-function get_pseudo_keycode(
+function get_pseudo_key(
     event: JQuery.KeyDownEvent | JQuery.KeyUpEvent | JQuery.KeyPressEvent,
-): number {
+): string {
     const isComposing = event.originalEvent?.isComposing ?? false;
-    /* We treat IME compose enter keypresses as a separate -13 key. */
-    if (event.keyCode === 13 && isComposing) {
-        return -13;
+    /* Ignore IME compose enter keypresses. (See 7 above) */
+    if (event.code === "Enter" && isComposing) {
+        return "IgnoreEnter";
     }
-    return event.keyCode;
+    return event.key;
 }
```

Comment:
This entire function can be replaced with `event.key`. For an Enter keydown during input method composition, browsers already give `event.key === "Process"`.

Reply:
Oh yeah!  - https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values

Updated!